TextClassifier.ipynb

```diff
@@ -0,0 +1,310 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Bolt TextClassifier Demo\n",
+    "\n",
+    "The bolt TextClassifier is an easy way to quickly develop, train, and test a model to perform classification and categorization of text data. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from thirdai import bolt\n",
+    "import datasets\n",
```

Comment:
Should we add an optional cell to pip install dependencies like datasets or is it safe to assume that this will already be done?

Reply:
good idea